### DISCUSSION TITLE
newer FIrmware available for Waveshare 485 to ETH POE available

### DISCUSSION THREAD

**Discussion by DerRonnyUK** [nathanmarlor/foxess_modbus#634] [General]
thought providing the latest FW from the manufacturer 
https://github.com/nathanmarlor/foxess_modbus/wiki/Waveshare-RS485-to-ETH-(B)

https://www.waveshare.com/wiki/RS485_TO_ETH_B_burns_firmware
its 1.490

**Comment by Ndrinta** [comment]
Thanks man, will add to the guide.
See here: [link to Guide](https://github.com/nathanmarlor/foxess_modbus/wiki/Waveshare-RS485-to-ETH-%28B%29)

**Comment by Ndrinta** [comment]
Forgot to ask, did they tell you what's different in this new version?

**Reply by DerRonnyUK** [reply]
sadly no - I just asked them for the latest FW - which is final and no beta or pre releases

**Comment by amalyomanh1** [comment]


**Reply by Ndrinta** [reply]
What?
